tests/test_cli_sync.py

```diff
@@ -242,3 +244,116 @@ def test_sync_dry_run_returns_non_zero_exit_code(runner):
     out = runner.invoke(cli, ["--dry-run"])
 
     assert out.exit_code == 1
+
+
+@mock.patch("piptools.scripts.sync.get_pip_version_for_python_executable")
+@mock.patch("piptools.scripts.sync.get_sys_path_for_python_executable")
+@mock.patch("piptools.scripts.sync.get_installed_distributions")
+@mock.patch("piptools.sync.run")
+def test_python_executable_option(
+    run,
+    get_installed_distributions,
+    get_sys_path_for_python_executable,
+    get_pip_version_for_python_executable,
+    runner,
+    fake_dist,
+):
+    """
+    Make sure sync command can run with `--python-executable` option
```

Comment:
PEP 257 requires sentences (with a period at the end)
```suggestion
    Make sure sync command can run with `--python-executable` option.
```

Reply:
updated here and in other places I found in my code